modules/account-lib/src/coin/polygon/transactionBuilder.ts

```diff
@@ -0,0 +1,42 @@
+import { BaseCoin as CoinConfig } from '@bitgo/statics/dist/src/base';
```

Comment:
Can we not import this from the root of the package?

Reply:
made the change